pkg/providers/amifamily/bottlerocket.go

```diff
@@ -128,17 +128,11 @@ func (b Bottlerocket) EphemeralBlockDevice() *string {
 // podsPerCore will be ignored
 // https://github.com/bottlerocket-os/bottlerocket/issues/1721
 
-// EvictionSoftEnabled is currently disabled for Bottlerocket AMIFamily because it does
-// not currently support the evictionSoft parameter passed through the kubernetes settings TOML userData
-// If a NodePool sets the evictionSoft value when using the Bottlerocket AMIFamily in the provider,
-// evictionSoft will be ignored
-// https://github.com/bottlerocket-os/bottlerocket/issues/1445
-
 func (b Bottlerocket) FeatureFlags() FeatureFlags {
 	return FeatureFlags{
 		UsesENILimitedMemoryOverhead: false,
 		PodsPerCoreEnabled:           false,
-		EvictionSoftEnabled:          false,
```

Comment:
I don't even think we need the feature flag at all now since I believe they are all `true` now. You can probably just drop the field outright from the struct now